File: cpp/Leetcode/Python/problem2149.py
class Solution:
    def rearrangeArray(self, nums):
        positive=[]
        negative=[]
        modified=[]
        for i in nums:
            if i>0:
                positive.append(i)
            elif i<0:
                negative.append(i)
        for i in range(int(len(nums)/2)):
            modified.append(positive[i])
            modified.append(negative[i])
        return modified
        

File: cpp/Leetcode/c++/problem2149.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    vector<int> rearrangeArray(vector<int>& nums) {
        vector<int>positive;
        vector<int>negative;
        vector<int>modified;
        for(auto i:nums){
            if(i>0){
                positive.push_back(i);
            }else if(i<0){
                negative.push_back(i);
            }
        }
        for(int i=0;i<nums.size()/2;i++){
            modified.push_back(positive[i]);
            modified.push_back(negative[i]);
        }
        return modified;
    }
};